web_app/web_app.py

```diff
@@ -135,9 +95,20 @@ def index():
             # modified_home_price = predict_home_price(bedrooms_new, bathrooms_new, sqft_new)
             modified_home_price = predict_home_price(bedrooms_new, bathrooms_new, sqft_new, stories_existing)
             tot_sqft = modified_sqft + additional_sqft
-            user_data = [bedrooms_addition, bathrooms_addition, kitchen, living_room, detached, modified_sqft, additional_sqft, second_story, tot_sqft]
-            samples = predict_cost_distribution(user_data, clf_mean, clf_sd, high_end)
-            
+            #user_data = [bedrooms_addition, bathrooms_addition, kitchen, living_room, detached, modified_sqft, additional_sqft, second_story, tot_sqft]
+            #samples = predict_cost_distribution(user_data, clf_mean, clf_sd, high_end)

```

Comment:
if these lines are no longer required, maybe consider removing them entirely from the file instead of commenting them out.